simple/utils/astrometry.py

```diff
@@ -357,3 +357,181 @@ def ingest_proper_motions(
             updated_source_pm_data.pprint_all()
 
     return
+
+
+def ingest_radial_velocity(
+    db: Database,
+    *,
+    source: str = None,
+    rv: Union[Quantity, float] = None,
+    rv_err: Optional[Union[float, Quantity]] = None,
+    reference: str = None,
+    raise_error: bool = True,
+):
+    """
+
+    Parameters
+    ----------
+    db: astrodbkit2.astrodb.Database
+        Database object
+    source: str
+        source name
+    rv: float or str
+        radial velocity of the sources
+        if not a Quantity, assumed to be in km/s
+    rv_err: float or str
+        radial velocity uncertainty
+        if not a Quantity, assumed to be in km/s
+    reference: str
+        reference for the radial velocity data
+    raise_error: bool
+        If True, raise errors. If False, log an error and return.
+
+    Returns
+    -------
+    flags: dict
+        'added' : bool
+        'skipped' : bool
+
+    Examples
+    ----------
+    > ingest_radial_velocity(db, my_source, rv=my_rv, rv_err=my_rv_unc,
+                             reference=my_rv_ref,
+                             raise_error = False)
```

Comment:
Example may be better with some actual numbers/strings as opposed to variable names to demonstrate the types of inputs (similar to what's in the tests).